src/module/actor/sheet/base.ts

```diff
@@ -190,15 +186,6 @@ export abstract class ActorSheetPF2e<TActor extends ActorPF2e> extends ActorShee
         );
     }
 
-    /** Save any open tinyMCE editor before closing */
-    override async close(options: { force?: boolean } = {}): Promise<void> {
-        const editors = Object.values(this.editors).filter((editor) => editor.active);
-        for (const editor of editors) {
-            editor.options.save_onsavecallback();
-        }
-        await super.close(options);
-    }
```

Comment:
does this not do anything for biographies?

Reply:
It's a holdover from when actor sheets weren't configured to save on close